packages/acceptit-Core.package/ACLibrary.class/class/generateNewLibrary..st

```diff
@@ -10,7 +9,7 @@ generateNewLibrary: aClassOrSymbol
 	
 	newClassName := (aClassOrSymbol asString, 'Library') asSymbol.
 	newCategory := (oldClass == Object)
-		ifTrue: [self libraryCategoryName] 
+		ifTrue: ['Generated'] 
```

Comment:
Why is this here? The branch is rebased to the newest version of develop I don't understand how those change revertions are in this branch....